src/SymbolicsUnitTests/Tests.fs

```diff
@@ -847,3 +856,12 @@ let ``Pseudo Function Test`` () =
     Infix.parseOrUndefined "pow(3*x,10*sin(x))" ==> "(3*x)^(10*sin(x))"
     Infix.parseOrUndefined "sqrt(pow(x,1/2))" ==> "(x^(1/2))^(1/2)"
 
+[<Test>]
+let ``Underscores in names`` () =
+    let expr = Infix.parseOrUndefined "(TESTING_UNDER)*(2)" 
+    expr ==> "2*TESTING_UNDER"
+    LaTeX.format expr --> """2{TESTING_{UNDER}}""" 
+
+    let expr2 = Infix.parseOrUndefined "(TESTING_UNDER_second)*(2)" 
+    expr2 ==> "2*TESTING_UNDER_second"
+    LaTeX.format expr2 --> """2{TESTING_{UNDER_{second}}}""" 
```

Comment:
I wonder whether this should be `{TESTING_{UNDER_{second}}}` or rather `{TESTING}_{UNDER}_{second}`. I'll have to check what renders correctly (unless you already know?).